pkgs/development/libraries/minc2-simple/default.nix

```
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, cmake, libminc }:
+
+stdenv.mkDerivation {
+  version = "unstable-2018-12-20";
+  name    = "minc2-simple";
+
+  src = fetchFromGitHub {
+    owner  = "vfonov";
+    repo   = "minc2-simple";
+    rev    = "e04414eb74052d8a6f9fb4bedda7f0894e86ac82";
+    sha256 = "1miy942ibbcikg0k45d7342ags2gpr29rvhvspwxqgv2jnpcwg05";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ libminc ];
+
+  cmakeFlags = [ "-DBUILD_TESTING=OFF" "-DLIBMINC_DIR=${libminc}/lib/" ];
+
+  # no cmake 'install' target
+  installPhase = ''
+    mkdir -p $out/lib $out/include
```

if you're going to override a phase, you should also add back the runhooks, incase someone one wants to override them
```suggestion
    runHook preInstall
    
    mkdir -p $out/lib $out/include
```